Apps/Common/src/AspNetConfiguration/StartupConfiguration.cs

```diff
@@ -233,11 +233,11 @@ public void UseForwardHeaders(IApplicationBuilder app)
                     RequireHeaderSymmetry = false,
                     ForwardLimit = null,
                 };
-                foreach (string ip in proxyIPs)
-                {
-                    options.KnownProxies.Add(IPAddress.Parse(ip));
-                }
 
+                // foreach (string ip in proxyIPs)
+                // {
+                //     options.KnownProxies.Add(IPAddress.Parse(ip));
+                // }
```

Comment:
if its not necessary lets remove commented out code

Reply:
I haven't made the determination yet... likely will be removed but this is still major flux